torch/_subclasses/fake_tensor.py

```diff
@@ -169,7 +216,12 @@ def check_non_fake_tensor(x):
         r = run_function(func, types, (), new_kwargs)
         return converter(r, out_device)
 
-    r = run_function(func, types, args, kwargs)
+    try:
+        r = run_function(func, types, args, kwargs)
+    except NotImplementedError as not_implemented_error:
+        if not cpu_fallback_enabled:
+            raise not_implemented_error
```

Comment:
`raise` here is better as doing it this way will destroy the original backtrace info

Reply:
sorry, how is this different from one i'm doing now ?